src/common/Navigation/index.js

```diff
@@ -1,9 +1,44 @@
+import React from "react";
+import {
+    StyledNavigation,
+    NavigationList,
+    StyledTitle,
+    StyledNavLink,
+    StyledNavWrapper,
+    StyledInput,
+    StyledLabel,
+    InputIcon,
+    StyledCameraIcon,
+} from "./styled";
+import { toActorsList, toMoviesList } from "../../core/config/routes";
+import cameraIcon from "./assets/images/CameraIcon.svg";
+import search from "./assets/images/SearchIcon.svg";
+
+
 const Navigation = () => {
     return (
-        <>
-            Navigation
-        </>
-    );
+        <StyledNavigation>
+            <StyledNavWrapper>
+                <StyledCameraIcon src={cameraIcon} />
+                <StyledTitle>Movies Browser</StyledTitle>
```

Comment:
StyledCameraIcon i StyledTitle powinny być obok siebie. Możesz jakiś wrapper wrzucić

Reply:
w przeglądarce są obok siebie, a u Ciebie się nie wyświetla kamerka obok tutłu?

Reply:
a dobra, flex-grow załatwia sprawę

Reply:
SVG można używać jako komponentów Reactowych musisz wtedy zrobić np
```js
import { ReactComponent as CameraIcon } from "../sciezka/do/obrazka.svg";
```
i wtedy wystarczy że wstawisz obrazek zamiast <img> a dodatkowo możesz to wrzucić do styled-components i wtedy ostylować np tak
```js
import { ReactComponent as CameraIconSVG } from "../sciezka/do/obrazka.svg";
import styled from "styled-components";

const CameraIcon = styled(CameraIconSVG)```
color: #FFFFFF;
width: 32px;
```;

nie mniej Twoja wersja też działa.

Innym problemem jest to, że ten obrazek kamery jest o wiele mniejszy niż na Figmie i coś tam się nie zgadza z pozycjami elementów względem tego szablonu